A32NX/layout.json

```diff
@@ -1,1019 +1,1029 @@
-{
```

Comment:
We have CI for this, no need to remove it but in the future no need to commit it